lib/beaker/options/parser.rb

```diff
@@ -32,7 +25,7 @@ def repo
       # Returns a description of Beaker's supported arguments
       # @return [String] The usage String
       def usage
-       @command_line_parser.usage
+        @command_line_parser.usage
```

Comment:
Not for this PR, as beaker is littered with this, but raising awareness (at more than a "style guide" level, as it has fairly deep practical implications): I recommend almost always using accessor methods (whether defined by `attr_*` or defined explicitly) instead of instance variable references (obvious exceptions include when the reference is inside an accessor method definition, when setting initially in a constructor, and so on).

The value of going this route shows up when refactoring, deprecating, auditing/instrumenting access, subclassing, etc., as instance variables work counter to these efforts. I've seen this as a "best practice" in smalltalk and java as well for similar reasons.

Again, not something to change here, just taking the opportunity to have the discussion.